src/main/java/programminglife/gui/controller/GuiController.java

```diff
@@ -578,6 +615,129 @@ void setText(int center, int radius) {
+        Text childText = new Text("Children: "); childText.setLayoutX(x); childText.setLayoutY(145);
+        Text inEdgeText = new Text("Incoming Edges: "); inEdgeText.setLayoutX(x); inEdgeText.setLayoutY(185);
+        Text outEdgeText = new Text("Outgoing Edges: "); outEdgeText.setLayoutX(x); outEdgeText.setLayoutY(225);
+        Text seqLengthText = new Text("Sequence Length: "); seqLengthText.setLayoutX(x); seqLengthText.setLayoutY(265);
+        Text genomeText = new Text("Genomes: "); genomeText.setLayoutX(x); genomeText.setLayoutY(305);
+        Text seqText = new Text("Sequence: "); seqText.setLayoutX(x); seqText.setLayoutY(370);
+
+        anchorGraphInfo.getChildren().removeIf(node1 -> node1.getLayoutX() == x);
+
+        TextField idTextField = getTextField("ID: ", x, 70, Integer.toString(node.getIdentifier()));
+
+        StringBuilder parentSB = new StringBuilder();
+        node.getParents().forEach(id -> parentSB.append(id).append(", "));
+        TextField parents;
+        if (parentSB.length() > 2) {
+            parentSB.setLength(parentSB.length() - 2);
+            parents = getTextField("Parents: ", x, 110, parentSB.toString());
+        } else {
+            parentSB.replace(0, parentSB.length(), "This node has no parent(s)");
+            parents = getTextField("Parents: ", x, 110, parentSB.toString());
+        }
+
+        StringBuilder childSB = new StringBuilder();
+        node.getChildren().forEach(id -> childSB.append(id).append(", "));
+        TextField children;
+        if (childSB.length() > 2) {
+            childSB.setLength(childSB.length() - 2);
+            children = getTextField("Children: ", x, 150, childSB.toString());
+        } else {
+            childSB.replace(0, childSB.length(), "This node has no child(ren)");
+            children = getTextField("Children: ", x, 150, childSB.toString());
+        }
+
+        String genomesString = graphController.getGraph().getGenomeNames(node.getGenomes()).toString();
+        String sequenceString = node.getSequence().replaceAll("(.{24})", "$1" + System.getProperty("line.separator"));
+        TextField inEdges = getTextField("Incoming Edges: ", x, 190, Integer.toString(node.getParents().size()));
+        TextField outEdges = getTextField("Outgoing Edges: ", x, 230, Integer.toString(node.getChildren().size()));
+        TextField seqLength = getTextField("Sequence Length: ", x, 270, Integer.toString(node.getSequence().length()));
+        TextArea genome = getTextArea("Genome: ", x, 310, genomesString.substring(1, genomesString.length() - 1), 40);
+        genome.setWrapText(true);
+        TextArea seq = getTextArea("Sequence: ", x, 375, sequenceString, 250);
+        anchorGraphInfo.getChildren().addAll(idText, parentText, childText, inEdgeText,
+                outEdgeText, genomeText, seqLengthText, seqText);
+        anchorGraphInfo.getChildren().addAll(idTextField, parents, children, inEdges, outEdges, genome, seqLength, seq);
+    }
```

Comment:
Yup. Honestly I couldn't care at that moment of doing it better since other things had more importance of finishing